.vscode/extensions.json

```diff
@@ -3,6 +3,7 @@
         "koihik.vscode-lua-format",
         "jkiviluoto.tws",
         "sumneko.lua",
-        "wwm.better-align"
+        "wwm.better-align",
+        "tomblind.local-lua-debugger-vscode"
     ]
 }
```

Comment:
Should have a new line.

Reply:
VSCode in general strips newlines from these json's when saving, it's one of the few exceptions we make in that regard in LSB (same as below)

Reply:
Oh great to know! Just wanted to make sure we are at least trying to follow CIs haha.